src/test/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCountsIntegrationTest.java

```diff
@@ -70,37 +69,23 @@ public void test(final File inputBAMFile,
         final File outputFile = createTempFile("collect-allelic-counts-test-output", ".tsv");
         final String[] arguments = {
                 "-" + StandardArgumentDefinitions.INPUT_SHORT_NAME, inputBAMFile.getAbsolutePath(),
-                "-" + ExomeStandardArgumentDefinitions.SITES_FILE_SHORT_NAME, SITES_FILE.getAbsolutePath(),
+                "-L", SITES_FILE.getAbsolutePath(),
                 "-" + StandardArgumentDefinitions.REFERENCE_SHORT_NAME, REF_FILE.getAbsolutePath(),
                 "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputFile.getAbsolutePath()
         };
         runCommandLine(arguments);
         final AllelicCountCollection countsResult = new AllelicCountCollection(outputFile);
+        Assert.assertEquals(countsExpected.getCounts().size(), countsResult.getCounts().size());
         Assert.assertEquals(countsExpected, countsResult);
     }
 
     //Regression test for https://github.com/broadinstitute/gatk-protected/issues/373
-    @Test(expectedExceptions = UserException.class)
-    public void testNonStrictBAM() {
-        final File outputFile = createTempFile("collect-allelic-counts-test-output", ".tsv");
-        final String[] arguments = {
-                "-" + StandardArgumentDefinitions.INPUT_SHORT_NAME, NON_STRICT_BAM_FILE.getAbsolutePath(),
-                "-" + ExomeStandardArgumentDefinitions.SITES_FILE_SHORT_NAME, SITES_FILE.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.REFERENCE_SHORT_NAME, REF_FILE.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputFile.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.READ_VALIDATION_STRINGENCY_SHORT_NAME, ValidationStringency.STRICT.toString()
-        };
-        runCommandLine(arguments);
-        //should catch SAMFormatException and throw new UserException with --readValidationStringency STRICT
-    }
-
-    //Regression test for https://github.com/broadinstitute/gatk-protected/issues/373
     @Test
```

Comment:
The purpose of these tests was to check that given a non-strict SAM test file the tool would run with -VS SILENT and throw an exception with -VS STRICT.  That doesn't seem to be the case now with the BAM you've created from the original SAM test file.  The original SAM file contained a single read from a contig not in the sequence dictionary; your BAM file turns this into an unmapped read with `RNAME=*`.

The original intent was to make sure the tool is well behaved for any validation stringency when given non-strict input, so I'll leave it to you to decide if that is true and covered by the current tests (perhaps @cmnbroad can chime in).

If you decide to remove this test, then remember to remove the simple-overhang test files as well.